ckan/requirements.txt

```diff
@@ -1,42 +1,42 @@
--i https://pypi.org/simple
--e git+https://github.com/GSA/ckanext-datajson.git@46ed53e6bfa3ad1f60bda06daba8fdf77138b254#egg=ckanext-datajson
--e git+https://github.com/ckan/ckan.git@7900935de6270ee80ec64a5378251dafedf3a0e4#egg=ckan
 argparse==1.4.0
 asn1crypto==1.3.0
 babel==2.3.4
 beaker==1.11.0
 bleach==2.1.3
 boto==2.49.0
 cffi==1.12.3
+-e git+https://github.com/ckan/ckan.git@7900935de6270ee80ec64a5378251dafedf3a0e4#egg=ckan
+-e git+https://github.com/ckan/ckanext-archiver.git@4cb10ac9f96fc78bc7be4b3dee8fbc56049b2865#egg=ckanext-archiver
```

Comment:
Archiver relies, via `requirements.txt` on:
```
celery = "==3.1.25"
kombu = "==3.0.37"
progressbar = "==2.3"
```

Just want to ensure that these requirements will be included. (I don't see them in the lockfile)

Reply:
I added these and updated the ckan/requirements.txt file.